radis/tools/line_survey.py

```diff
@@ -267,61 +267,80 @@ def get_label_hitran(row, details, attrs):
             molecule = get_molecule(attrs["id"])
 
         else:
-            molecule = get_molecule(row.id)
+            id = get_molecule_identifier(spec.conditions["molecule"])
+            molecule = get_molecule(id)
 
         # Get global labels
         if molecule in HITRAN_CLASS1:
+
+            add = ["vu", "vl", "jl", "iso"]
+
+            if "isotope" in spec.conditions:
+                iso = spec.conditions["isotope"]
+                add.append("iso")
```

Comment:
warning, "iso" is already in add here ?

Reply:
Ya right !!

Reply:
I have a few test in `test_calc.py` that seem to be passing on travis but are failing locally (after the merge from develop ). Any idea of what might be going wrong?

Reply:
Some tests are excluded on Travis ; mostly those requiring large databases. Default Travis config is defined [here](https://github.com/radis/radis/blob/develop/setup.cfg#L12).  You may get these defaults if you run pytest from the level root; but not if you're in a folder.